canonical-web-and-design/juju.is
# Build the about high-availability-enterprise-olm page

## QA
- Go to http://0.0.0.0:8041/about/high-availability-enterprise-olm
- Check the content matches the [copy doc](https://docs.google.com/document/d/1hyW0MSqIc9bS9NMcE6VPpX5f2PAksLUBIC2oU6IRMl0/edit#)

Fixes https://github.com/canonical-web-and-design/juju.is/issues/106
